scout/server/blueprints/clinvar/templates/clinvar_submissions.html

```diff
@@ -137,7 +137,7 @@ <h4>Variant data:</h4>
               <tr>
                 <td>{{loop.index}}</td>
                   {% if subm_variant.category == 'sv' %}
-                    {% do var_key_name.update( {subm_variant.local_id : '_'.join([subm_variant.chromosome, subm_variant.breakpoint1, subm_variant.ref[0:5], subm_variant.alt[0:5]]) }) %}
+                    {% do var_key_name.update( {subm_variant.local_id : '_'.join(["chr", subm_variant.chromosome, subm_variant.var_type]) }) %}
```

Comment:
Hm, wouldn't that break for like a dup-inv-dup or such where you have two of the same kind nearby? A position as well perhaps?

Reply:
Sure, let's add it!

Reply:
Fixed it like this, in case the variant doesn't have breakpoints but only inner and outer coordinates:

<img width="488" alt="image" src="https://user-images.githubusercontent.com/28093618/205305381-7196f371-c010-46ef-b385-7eaea0001595.png">

While fixing this I've realized that also inner and outer coords (start and stop) were not saved in the submission object. So I've fixed this as well!